velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -921,6 +921,21 @@ std::optional<std::string> getTimestampTimeZone(
   return std::nullopt;
 }
 
+parquet::WriterOptions::DataPageVersion getParquetDataPageVersion(
```

Comment:
The changes in this file have to move to the `parquet::WriterOptions` inside `parquet/writer/Writer.h`

Reply:
Done.